resources/assets/sass/regions/_home.scss

```diff
@@ -0,0 +1,284 @@
+.r-home {
+
+    background: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.